input.html

```diff
@@ -5,8 +5,8 @@
   <style is="custom-style">
       html {
           --kano-input: {
-              font-family: bariol;
-              font-size: 1.1em;
+              font-family: var(--font-body);
```

Comment:
If relying on the var, you can import the `typography` file to make sure bundlers see the real tree of dependencies

Reply:
Yep, the import is there – I've updated the spacing to make it a bit clearer